src/main/scala/org/broadinstitute/dsde/firecloud/service/DataUseRestrictionSupport.scala

```diff
@@ -2,16 +2,40 @@ package org.broadinstitute.dsde.firecloud.service
 
 import com.typesafe.scalalogging.LazyLogging
 import org.broadinstitute.dsde.firecloud.dataaccess.OntologyDAO
-import org.broadinstitute.dsde.firecloud.model.DUOS.DuosDataUse
+import org.broadinstitute.dsde.firecloud.model.DUOS.{DuosDataUse, StructuredDataRequest, StructuredDataResponse}
 import org.broadinstitute.dsde.firecloud.model.DataUse.DiseaseOntologyNodeId
 import org.broadinstitute.dsde.rawls.model.Attributable.AttributeMap
 import org.broadinstitute.dsde.rawls.model.WorkspaceJsonSupport.AttributeNameFormat
-import org.broadinstitute.dsde.rawls.model._
+import org.broadinstitute.dsde.rawls.model.{Attribute, _}
 import spray.json.DefaultJsonProtocol._
 import spray.json._
 
 import scala.util.Try
 
+object ConsentCodes extends Enumeration {
```

Comment:
You might be able to make better use of `Enumeration`. If you define the values like:
```
val GRU = Value("GRU")
```
and add a function here:
```
  def withNameIgnoreCase(s: String): Value =
    values.find(_.toString.equalsIgnoreCase(s)).getOrElse(throw new NoSuchElementException(s"No value found for '$s'"))
```
then you can match against the enum values:
```
ConsentCodes.withNameIgnoreCase(attr.name) match {
  case ConsentCodes.NAGR => ...
```
The downside is that you have to use `toString` everywhere to get the string value out.

Alternately, you can just remove the `extends Enumeration` here and leave it as a bag of string constants.